route/build/src/mpi_process.f90

```diff
@@ -1154,6 +1199,8 @@ SUBROUTINE mpi_route(pid,           & ! input: proc id
   USE globalData, ONLY: basinEvapo_trib     ! tributary only HRU Evaporation
   USE globalData, ONLY: basinPrecip_main    ! mainstem only HRU Precipitation
   USE globalData, ONLY: basinPrecip_trib    ! tributary only HRU Precipitation
+  USE globalData, ONLY: basinCC_main        ! mainstem only HRU consitunet mass
```

Comment:
Misspelling and mention solute again.

Reply:
change `basinCC_main` to `basinSolute_main`, then `consituent` to `solute`